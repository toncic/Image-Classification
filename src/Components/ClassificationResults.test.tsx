import React from "react";
import renderer from "react-test-renderer";
import ClassificationResults from "./ClassificationReults";

const exampleData = [
  { className: "test1", probability: "0.3333" },
  { className: "test2", probability: "0.4444" },
  { className: "test3", probability: "0.5555" }
];

it("classification result are shown properly", () => {
  const component = renderer.create(
    <ClassificationResults results={exampleData} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
