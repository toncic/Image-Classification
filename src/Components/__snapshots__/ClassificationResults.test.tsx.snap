// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`classification result are shown properly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-28 MuiPaper-rounded"
  >
    <h3
      className="MuiTypography-root MuiTypography-h5"
    >
      test1
    </h3>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      0.3333
    </p>
     
  </div>
  <div
    className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-28 MuiPaper-rounded"
  >
    <h3
      className="MuiTypography-root MuiTypography-h5"
    >
      test2
    </h3>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      0.4444
    </p>
     
  </div>
  <div
    className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-28 MuiPaper-rounded"
  >
    <h3
      className="MuiTypography-root MuiTypography-h5"
    >
      test3
    </h3>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      0.5555
    </p>
     
  </div>
</div>
`;
